tests/test_api.py

```diff
@@ -69,6 +72,6 @@ def test_stream(server):
 
 
 @pytest.mark.asyncio
-async def test_get_invalid_url(server):
+async def test_get_invalid_url(server: Server) -> None:
     with pytest.raises(httpx.InvalidURL):
-        await httpx.get("invalid://example.org")
+        await httpx.get("invalid://example.org")  # type:ignore
```

Comment:
Ah, so this entire test should really not be an `async` one — the high-level API is sync-only.